test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -469,3 +469,66 @@ Feature: Bind values from a secret referred in backing service resource
         Then Service Binding "ssa-3" is ready
         And The application env var "BACKEND_USERNAME" has value "AzureDiamond"
         And The application env var "BACKEND_PASSWORD" has value "hunter2"
+
+    # Remove this disable tag once this issue is closed: https://github.com/redhat-developer/service-binding-operator/issues/808
+    @disabled
```

Comment:
we should see this failing in some CI run.

Reply:
okay.